### PR TITLE
nf-core modules bump-versions for all modules 

### PR DESCRIPTION
**Pull request by drpatelh**
Duplicate of https://github.com/nf-core/modules/pull/629 . Easier to re-run `nf-core modules bump-versions` in a separate PR.

* Added new `.nf-core.yml` file to ignore bumping modules where recent versions are known to be broken. These can be tested properly, fixed and removed from this file in the future.
* Ran `nf-core modules bump-versions --all` to update remainder of modules in one go!!! 🥳 
* Updates failed for the following modules:

```bash
╭───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [!] 26 Module updates failed                                                                                                                                                              │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
╭──────────────────────────────────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Module name                              │ Update message                                                                                                                                 │
├──────────────────────────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ bbmap/bbduk                              │ Could not download container tags: Unexpected response code 204 for                                                                            │
│                                          │ https://api.biocontainers.pro/ga4gh/trs/v2/tools/bbmap/versions/bbmap-38.91                                                                    │
│ bismark/report                           │ Could not download container tags: Could not find docker or singularity container for bismark                                                  │
│ bismark/align                            │ Could not download container tags: Could not find docker or singularity container for bismark                                                  │
│ bismark/genomepreparation                │ Could not download container tags: Could not find docker or singularity container for bismark                                                  │
│ bismark/summary                          │ Could not download container tags: Could not find docker or singularity container for bismark                                                  │
│ bismark/deduplicate                      │ Could not download container tags: Could not find docker or singularity container for bismark                                                  │
│ bismark/methylationextractor             │ Could not download container tags: Could not find docker or singularity container for bismark                                                  │
│ bowtie/align                             │ Ignoring mulled container                                                                                                                      │
│ bowtie2/align                            │ Ignoring mulled container                                                                                                                      │
│ bwa/mem                                  │ Ignoring mulled container                                                                                                                      │
│ bwamem2/mem                              │ Ignoring mulled container                                                                                                                      │
│ cat/fastq                                │ Ignoring mulled container                                                                                                                      │
│ dshbio/splitgff3                         │ Could not download container tags: Unexpected response code 204 for                                                                            │
│                                          │ https://api.biocontainers.pro/ga4gh/trs/v2/tools/dsh-bio/versions/dsh-bio-2.0.5                                                                │
│ dshbio/splitbed                          │ Could not download container tags: Unexpected response code 204 for                                                                            │
│                                          │ https://api.biocontainers.pro/ga4gh/trs/v2/tools/dsh-bio/versions/dsh-bio-2.0.5                                                                │
│ dshbio/filterbed                         │ Could not download container tags: Unexpected response code 204 for                                                                            │
│                                          │ https://api.biocontainers.pro/ga4gh/trs/v2/tools/dsh-bio/versions/dsh-bio-2.0.5                                                                │
│ dshbio/filtergff3                        │ Could not download container tags: Unexpected response code 204 for                                                                            │
│                                          │ https://api.biocontainers.pro/ga4gh/trs/v2/tools/dsh-bio/versions/dsh-bio-2.0.5                                                                │
│ gunzip                                   │ Ignoring mulled container                                                                                                                      │
│ hifiasm                                  │ Could not download container tags: Could not find docker or singularity container for hifiasm                                                  │
│ hisat2/align                             │ Ignoring mulled container                                                                                                                      │
│ kraken2/kraken2                          │ Did not find pattern quay.io/biocontainers/kraken2:[^'"\s]+ in module kraken2/kraken2                                                          │
│ rsem/calculateexpression                 │ Ignoring mulled container                                                                                                                      │
│ rsem/preparereference                    │ Ignoring mulled container                                                                                                                      │
│ seacr/callpeak                           │ Ignoring mulled container                                                                                                                      │
│ star/genomegenerate                      │ Ignoring mulled container                                                                                                                      │
│ untar                                    │ Ignoring mulled container                                                                                                                      │
│ yara/mapper                              │ Ignoring mulled container                                                                                                                      │
╰──────────────────────────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

### CODE REVIEW

**Comment by heuermh** [comment]
In https://github.com/nf-core/modules/pull/628 for `dsh-bio` 2.0.5 conda and docker were ok but the singularity image was not yet available

File: modules/iqtree/main.nf [hunk 1]
```diff
@@ -11,11 +11,11 @@ process IQTREE {
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:[:], publish_by_meta:[]) }
 
-    conda (params.enable_conda ? "bioconda::iqtree=2.1.2" : null)
+    conda (params.enable_conda ? 'bioconda::iqtree=2.1.4_beta' : null)
     if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://depot.galaxyproject.org/singularity/iqtree:2.1.2--h56fc30b_0"
+        container "https://depot.galaxyproject.org/singularity/iqtree:2.1.4_beta--hdcc8f71_0"
```
Comment (JoseEspinosa) [inline modules/iqtree/main.nf:16]: For the naming this one seems to be a beta, do we want to update it?

File: modules/prodigal/meta.yml [hunk 2]
```diff
@@ -1,32 +1,27 @@
 name: prodigal
-## TODO nf-core: Add a description of the module and list keywords
-description: write your description here
+description: Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm) is a microbial (bacterial and archaeal) gene finding program
```
Comment (JoseEspinosa) [inline modules/prodigal/meta.yml:2]: Might be cleaner to keep this PR clean with only version changes

Comment (JoseEspinosa) [reply modules/prodigal/meta.yml:2]: BTW, after writing the comment it sounds really very @drpatelh -ish, am I wrong? :smile:

**Review by JoseEspinosa — APPROVED** [review]
Just a question about a module that seems to be updated to beta, but otherwise, feel free to merge when tests are passing... or before :stuck_out_tongue:

File: modules/prodigal/meta.yml (hunk 2 above)
Comment (drpatelh) [reply modules/prodigal/meta.yml:2]: Nope. My bad. Should have done it in a separate PR. You are right. Trying to do 1000 things so squeezed it in here 😬

Comment (drpatelh) [reply modules/prodigal/meta.yml:2]: These came up when I was testing `nf-core lint` which is why I fixed them.

File: modules/iqtree/main.nf (hunk 1 above)
Comment (drpatelh) [reply modules/iqtree/main.nf:16]: Tests seem to be passing 🤷🏽 It can be downgraded manually later if required.

**Pull request merged**